update-repo.sh

```diff
@@ -16,20 +16,28 @@ aports_dir=aports/ros/$1
 # env vars:
 #   APORTS_SLUG_UPSTREAM
 #   APORTS_SULG
-#   PACKAGE_LIST
 #   ROS_DISTRO
 #   PARALLEL
 #   GIT_EMAIL
 
 aports_slug_upstream=${APORTS_SLUG_UPSTREAM:-seqsense/aports-ros-experimental}
 aports_slug=${APORTS_SLUG:-${aports_slug_upstream}}
 ros_distro=${ROS_DISTRO:-noetic}
+is_ros2=true
+if [ ${ros_distro} = "melodic" ] || [ ${ros_distro} = "noetic" ]; then
+  is_ros2=false
+fi
```

Comment:
I think giving `DISTRIBUTION_TYPE` as a build argument and explicitly specifying in matrix.env is simpler